pom.xml

```diff
@@ -1576,6 +1575,10 @@
                   <goal>npm</goal>
                 </goals>
                 <phase>pre-site</phase>
+                <configuration>
+                  <arguments>install github:apache/logging-parent#feature/366-npm-shrinkwrap</arguments>
+                  <!--<arguments>install github:apache/logging-parent#rel/${stable-project.version}</arguments>-->
```

Comment:
I presume this will be fixed before merging this PR?

Reply:
Fixed in https://github.com/apache/logging-parent/pull/367/commits/5cb465e7dfa9cff6cad5d55ace0a775c8ee75eef